mailchimp/wordpress
# Update the svg that we use in our menu item

### Description of the Change

As reported in #63, the svg used in the Mailchimp menu is the wrong one (basically the colors were inverted). This PR fixes that.

**Before:**

![menu svg before](https://github.com/user-attachments/assets/e95a114f-2ea6-41e1-90e1-8851b05f9042)

**After:**

![menu svg after](https://github.com/user-attachments/assets/5d6ded10-7dda-4cd1-bcaa-e3d5f0309991)

Closes #63

### How to test the Change

1. Check out these changes
2. View the icon that shows in the left-hand sidebar and ensure it looks correct

### Changelog Entry

> Update - Updated the menu SVG icon.

### Credits

Props @dkotter

### Checklist:
